src/game/game-settings.ts

```diff
@@ -17,8 +18,17 @@ export interface GameSettings {
     PLAYER_INITIAL_ENERGY: number;
     PLAYER_MAX_ENERGY: number;
     // AI Settings
-    AI_MAX_SHOOT_DELAY: number;
-    AI_MIN_SHOOT_DELAY: number;
+    AI_DUMBNESS: number[]; // For each possible player, value from 0..1 where 0 is minimal dumbness and 1 is maximum dumbness
+    AI_SHOOT_DELAY: number;
+    AI_HEALTH_SCORE_WEIGHT: number; // how much it is priority to target weak players
+    AI_HEALTH_POWERUP_SCORE_WEIGHT: number; // how much it is priority to target healing powerups
+    AI_POWERUP_ACTIONABLE_RADIUS: number;
+    AI_POWERUP_IGNORE_DELAY: number;
+    AI_MAX_AIM_ERROR: number; // Maximal radial difference from aim angle, decreases with growing delay between shots
+    AI_AIM_ERROR_DECREASE_SPEED: number;
+    AI_MAX_COLLISION_DODGE_ERROR: number; // How long AI can wait (before it "notices the collison") before it starts evading
+    AI_COLLISION_DODGE_ERROR_DECREASE_SPEED: number;
+    AI_MAX_COLLISION_PANIC_RADIUS: number; // AI can "panic" when computing incoming collisions, by enlarging myPlayer radius by maximally this amount
```

Comment:
IMHO This should be configurable per AI Brain, but we can do that later.

Reply:
AI_DUMBNESS actually dictates how much most of the remaining values should be taken into account.